### ISSUE TITLE
Add `ContentRemovalRequest` viewset

### ISSUE THREAD

**Issue by bjester** [learningequality/kolibri#10381]
Depends on: https://github.com/learningequality/kolibri/pull/10365

## Background
In order to provide support for managing a user's 'My Downloads', we need a new API viewset that will allow us manage removal of content.

### The `ContentRemovalRequestViewSet`
a) It should inherit `ValuesViewset`
b) It should operate upon the model `ContentRemovalRequest`

### The `ContentRemovalRequestViewSet`: request creation
a) It should allow creation of a `ContentRemovalRequest` model for a user
b) It should create the model with the appropriate fields. [See `ContentRequest.build_for_user`](https://github.com/learningequality/kolibri/pull/10365/files#diff-13c98dfd936c37246a4bc23043dd6f6eb5a8a04d26e185d8c771c8e946fa407cR455-R468)

## Acceptance criteria
Appropriate tests for the viewset

## References
https://github.com/learningequality/kolibri/pull/10137

**Comment by rtibbles** [comment]
This has been folded into #10379 in order to give a more simplified frontend API interaction experience.
